scripts/release/bump_version.js

```diff
@@ -14,6 +14,31 @@ const convertPrereleaseType = (version, prereleaseType) => {
   return `${semver.major(version)}.${semver.minor(version)}.0-${prereleaseType}.1`;
 };
 
+const handlePackages = (versionFunc, releaseType) => {
```

Comment:
Why not to do 2 functions? It will be cleaner to have two packagesIterator for rust and js as well